app/assets/stylesheets/katello/contents.scss

```diff
@@ -1,5 +1,5 @@
 @import "katello/katello_base";
-@import "jquery.timepickr";
+@import "katello/jquery.timepickr";
```

Comment:
You might could even remove this -- I am not sure where timepickr would be used in the old UI